src/components/GetAllComments.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { getComment } from "../../websiteAPI";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+import ArticleCard from "./ArticleCard";
+import { getArticle } from "../../websiteAPI";
+import NewComment from "./NewComment";
+
+const GetAllComments = ({reqArticle, currVote, setReqArticle, username}) => {
```

Comment:
GetAllComments seems like the wrong component name if the component also gets the article!